test.css

```diff
@@ -1,386 +1,394 @@
-*{
-    /*font-family: 'Overpass', sans-serif;*/
-    font-family: 'Roboto', sans-serif;
-    font-size: 15px;
-}
-.header{
-    width: 100vmax;
-    position: relative;
-    z-index: 1000;
-    background-color: rgb(255, 255, 255);
-}
-nav{
-    display: flex;
-}
-.menu{
-    flex: 1;
-    text-align: left;
-}
-.menu ul{
-    margin-top: 20px;
-    margin-left: 10px;
-}
-.menu ul li{
-    list-style: none;
-    display: inline-block;
-}
-.menu ul li a{
-    text-decoration: none;
-    display: inline-block;
-    color: #7F7E8A;
-    border: 1px transparent;
-    border-radius: 8px;
-    padding: 5px;
-    position: relative;
-    background: transparent;
-    cursor: pointer;
-    
-}
-.sign_menu ul{
-    margin-right: 20px;
-    margin-top: 5px;
+* {
+  /*font-family: 'Overpass', sans-serif;*/
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+}
+header {
+  width: 100%;
+  position: relative;
+  z-index: 1000;
+  background-color: rgb(255, 255, 255);
+}
+nav {
+  display: flex;
+}
+.menu {
+  flex: 1;
+  text-align: left;
+}
+.menu ul {
+  margin-top: 20px;
+  margin-left: 10px;
+}
+.menu ul li {
+  list-style: none;
+  display: inline-block;
+}
+.nav_btn {
+  text-decoration: none;
+  display: inline-block;
+  color: #7f7e8a;
+  border: 1px transparent;
+  border-radius: 8px;
+  padding: 5px;
+  position: relative;
+  background: transparent;
+  cursor: pointer;
+}
```

Comment:
Instead of using **.menu ul li a** when you have a class name **.nav_btn** already, use the class name directly to select.